packages/gestalt/src/ExpandableModule.js

```diff
@@ -0,0 +1,65 @@
+// @flow strict
+import React, { useState, type Node } from 'react';
+import Box from './Box.js';
+import Divider from './Divider.js';
+import ExpandableModuleBase from './ExpandableModuleBase.js';
+import icons from './icons/index.js';
+import styles from './ExpandableModule.css';
+
+type Props = {|
+  items: Array<{|
+    title: string,
+    icon?: $Keys<typeof icons>,
+    iconAccessibilityLabel?: string,
+    accessibilityLabel: string,
+    accessibilityControls: string,
+    summary?: Array<string>,
+    type?: 'error' | 'info',
+    children?: Node,
+  |}>,
+|};
+
+export default function ExpandableModule({ items }: Props): Node {
+  const [expandedId, setExpandedId] = useState(-1);
+
+  return (
+    <div className={styles.expandableModule}>
```

Comment:
@galaxy0101 this can now be `<Box rounding={2} borderStyle="shadow">`. We should no longer need any custom CSS for this component.

Reply:
Fixed.